### ISSUE TITLE
Segmentation fault

### ISSUE THREAD

**Issue by peyre** [raceintospace/raceintospace#687]
I was in the middle of a Docking (Orbit) mission and immediately after a successful docking (before it showed the movie for the next part) the game crashed showing "Segmentation fault (core dumped)" in the terminal.  I have the save game if anyone wants to see it.

**Comment by peyre** [comment]
On second thought, I think something else is malfunctioning.  I'm running a plain vanilla download of the game and still getting segfaults.  I just got one in the middle of an orbital mission.

**Comment by hweimer** [comment]
If you have a recent version of CMake, it might help to add `add_compile_options(-fsanitize=address)` to your top-level CMakeLists.txt to get more useful error messages.

**Comment by peyre** [comment]
I have 3.22.1 installed.

Now, I have CMakeLists.txt files _all over_ my computer, looks like hundreds of copies.  Do you mean the one in ~/raceintospace?

**Comment by hweimer** [comment]
Yes, if you checked out the repository into ~/raceintospace, that's where you should add the line to the CMakeLists.txt.

**Comment by peyre** [comment]
Sorry for the delay.  I encountered this again, saved the autosave, then modified my CMakeLists.txt and recompiled - but the mission the game segfaulted on won't segfault again!
